x-pack/legacy/plugins/uptime/server/lib/adapters/monitor_states/search/__tests__/query_context.test.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { QueryContext } from '../query_context';
+import { CursorPagination } from '../..';
+import { CursorDirection, SortOrder } from '../../../../../../common/graphql/types';
+
+describe(QueryContext, () => {
+  // 10 minute range
+  const rangeStart = '2019-02-03T19:06:54.939Z';
+  const rangeEnd = '2019-02-03T19:16:54.939Z';
+
+  const pagination: CursorPagination = {
+    cursorDirection: CursorDirection.AFTER,
+    sortOrder: SortOrder.DESC,
+  };
+
+  let qc: QueryContext;
+  const makeQueryContext = () => {
+    return new QueryContext({}, rangeStart, rangeEnd, pagination, null, 10);
+  };
+  beforeEach(() => (qc = makeQueryContext()));
```

Comment:
```suggestion
  beforeEach(() => (qc = new QueryContext({}, rangeStart, rangeEnd, pagination, null, 10)));
```

Could we just instantiate a new context directly here, since `makeQueryContext` isn't referenced anywhere else in the file?

Reply:
Yes, I left it there as a style preference, since I thought we might use it in the future, but I think you're right and YAGNI applies here.